src/main/kotlin/com/grappenmaker/optifineweave/WeaveOptifineURL.kt

```diff
@@ -85,9 +86,9 @@ class CapeHook : Hook("net/optifine/player/CapeUtils") {
 }
 
 @Suppress("unused")
-class LunarBridgeHook : Hook("*") {
+class LunarBridgeHook : Hook() {
     override fun transform(node: ClassNode, cfg: AssemblerConfig) {
-        if (!node.name.startsWith("com/moonsworth/")) return
+        if (!node.name.startsWith("com/moonsworth")) return
```

Comment:
Why

Reply:
I was having issues with the hook (I misunderstood something Nilsen said), and I must've removed that while messing around and forgot to re-add it. It doesn't change the functionality of the hook so it's fine to stay like that.